### PR TITLE
Sales engine ln

### PR DESCRIPTION
**Pull request by lnchambers**
Finished out basic framework and built out gitignore again. Don't break it this time.

### CODE REVIEW

File: test/fixtures/merchant_fixture.csv [hunk 1]
```diff
@@ -13,7 +13,7 @@ id,name,created_at,updated_at
 12, byMarieinLondon, 2016-01-11 11:59:37 UTC, 2015-07-28 04:52:03 UTC
 13, JUSTEmonsters, 2016-01-11 12:17:26 UTC, 1980-11-23 09:11:30 UTC
 14, Uniford, 1999-02-13 02:27:21 UTC, 2011-02-08 11:30:31 UTC
-15, thepurplepenshop, 2016-01-11 18:30:35 UTC, 2084-08-03 01:12:21 UTC
+15, Uniford, 2016-01-11 18:30:35 UTC, 2084-08-03 01:12:21 UTC
```
Comment (zpalmquist) [inline test/fixtures/merchant_fixture.csv:16]: wondering why we have two of these merchants? I think the more variance the better.

File: test/merchant_repo_test.rb [hunk 2]
```diff
@@ -8,13 +9,29 @@ def setup
...
+    assert_instance_of Merchant, merchant
+    assert_equal " jejum", merchant.name
+  end
+
+  def test_find_all_merchants_by_name
+    merchant = @merchants.find_all_by_name(" Uniford")
+
+    assert_equal 2, merchant.count
+    assert_equal " Uniford", merchant[0].name
+    assert_equal " Uniford", merchant[1].name
```
Comment (zpalmquist) [inline test/merchant_repo_test.rb:34]: aaah now I see. I'll allow it!

**Review by zpalmquist — APPROVED** [review]
Looks good man! I'll get to the testing and hopefully we can wrap up it0 early today and start off strong!

File: test/fixtures/merchant_fixture.csv (hunk 1 above)
Comment (lnchambers) [reply test/fixtures/merchant_fixture.csv:16]: I wanted to test if we got all the merchants by name.

**Pull request merged**